Customer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Voucherify;
+
+class Customer extends VoucherifyRequest {
+
+    /**
+     * Customer constructor.
+     *
+     * @param string $apiID
+     * @param string $apiKey
+     */
+    public function __construct($apiID, $apiKey) {
+        parent::__construct($apiID, $apiKey);
+    }
+
+    /**
+     * @param Customer $customer
+     *
+     * Create customer.
+     *
+     * @throws Voucherify\ClientException
+     */
+    public function create($customer) {
+        return $this->apiRequest("POST", "/customers/", NULL, $customer);
+    }
+
+    /**
+     * @param string $customerId
+     *
+     * Get customer details
+     *
+     * @throws Voucherify\ClientException
+     */
+    public function fetch($customerId) {
```

Comment:
@imvanzen  please change method name. must be consistent with other SDKs. Usse name `get`